dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -176,6 +176,12 @@ export const RenderPagination = (props) => {
 };
 
 export const EditRow = (props) => {
+  const isDirtyRow = () => {
+    return (
+      props.item[CONSTANTS.IS_DIRTY_NAME] ||
+      props.item[CONSTANTS.IS_DIRTY_SERVER_DELETE]
+    );
+  };
```

Comment:
This is nice, but it should probably be defined somewhere near where `markItemsDirty()` is defined, since they are (or, should be) intimately related.

(And, eventually, you'll probably want to define the `IS_DIRTY_*` symbols in that same place, and I expect that there will eventually be some more related code located there, as well.)